src/plugins/twitter/twitter.js

```diff
@@ -31,6 +31,61 @@ var componentName = "wb-twitter",
 			protocol = wb.pageUrlParts.protocol;
 
 		if ( eventTarget ) {
+
+			// Ignore IE11
+			// Note: Twitter's widget script doesn't render anything in IE11... so there's no value in showing an endless loading icon in it
+			if ( !wb.ie11 ) {
+				const linkSelector = "a.twitter-timeline",
+					twitterLinks = eventTarget.querySelectorAll( linkSelector ),
+					loadingClass = "twitter-timeline-loading";
+				let loadingRemovals = 0,
+					observer;
+
+				// Insert loading icons below Twitter links
+				twitterLinks.forEach( function( twitterLink ) {
+					const loadingDiv = document.createElement( "div" );
+					loadingDiv.className = loadingClass;
+
+					twitterLink.after( loadingDiv );
+				} );
+
+				// Remove loading icon after the timeline widget appears
+				// Note: Twitter's widget script removes "a.twitter-timeline" upon filling-in the timeline's content... at which point the loading icon is no longer useful
+				observer = new MutationObserver( function( mutations ) {
+
+					// Check for DOM mutations
+					mutations.forEach( function( mutation ) {
+
+						// Deal only with removed HTML nodes
+						mutation.removedNodes.forEach( function( node ) {
+							const twitterLink = node.matches( linkSelector );
+
+							// Continue if the removed node was a Twitter link
+							if ( twitterLink ) {
+								const linkSibling = mutation.nextSibling;
+
+								// Remove the link's adjacent loading icon
+								if ( linkSibling instanceof HTMLElement && linkSibling.matches( "." + loadingClass ) ) {
```

Comment:
Propose change:

```suggestion
								if ( linkSibling.classList && linkSibling.classList.contains( loadingClass ) ) {
```

Reply:
Good catch :)!

I dug around a bit and it looks like this suggestion would be simpler + more performant + more compatible (with older browsers) vs what I was doing.